### ISSUE TITLE
Pruning job enhancements

### ISSUE THREAD

**Issue by m-kratochvil** [tabernarious/f5-automated-backup-iapp#24]
#Feature requests - based on v3.1.9

It would be great if the pruning job could have following enhancements implemented:

- option to select the pruning interval (in our case 24h is desired)
- option to set the exact time of the first pruning job occurrence. Currently,  the pruning starts at the time the iApp is configured but it would be better if the pruning could be configured to happen  right after the backup is taken.

I know that the pruning interval can be adjusted within the template but it would be nicer to have that option as a variable, adjustable in the iApp configuration.

Thanks!
Michal Kratochvil

**Comment by tristanbob** [comment]
I totally support this improvement.  We discovered that pruning happens very 60 seconds when we were trying to review the audit log, only to find the log had thousands of entries from the pruning process.  We have changed pruning frequency in the template code to 1440 seconds (1 day), but this should be part of the iapp settings.  Thanks!

```
Wed Feb 19 03:12:00 MST 2020 |   | 0-0 | user=root action="periodic handler, run script: /Common/f5.automated_backup__Daily_Local_Backups_pruning" status="OK":
Wed Feb 19 03:13:00 MST 2020 |   | 0-0 | user=root action="periodic handler, run script: /Common/f5.automated_backup__Daily_Local_Backups_pruning" status="OK":
Wed Feb 19 03:14:00 MST 2020 |   | 0-0 | user=root action="periodic handler, run script: /Common/f5.automated_backup__Daily_Local_Backups_pruning" status="OK":
```

**Comment by tristanbob** [comment]
In the meantime, if anyone else is wondering how to change this, here is a snippet of the template that we edited.  You only need to change the "60" to be "1440".  Make sure to "reconfigure" your backup iapps to make the change take affect.

 ```
# Interval can be increased as needed if pruning every minute is problematic
iapp::conf create sys icall handler periodic f5.automated_backup__${tmsh::app_name}_pruning-handler \{ \
interval 1440 \
```

**Comment by tabernarious** [comment]
I know this is old but it will be available in v3.2.4, and I'm changing the default pruning interval to 86400 (once per day).

**Comment by tabernarious** [comment]
This has been fixed in v3.2.4, which I released today. Reopen this issue if you have further issues or requests.
